src/Modifiable.java

```diff
@@ -0,0 +1,7 @@
+public interface Modifiable {
+    int getId();         // Method to retrieve the ID
+    void setId(int id);  // Method to set the ID
+    void setTime(long time);
+    void setStadiumSector(char stadiumSector);
+    String getAllValues();
+}
```

Comment:
First of all, I would like to ask why `getAllValues()` in the `Modifiable `class is a `String`.
And secondly, I don’t see a reason to have this method here, do you?